src/scss/01_base/_variables.scss

```diff
@@ -74,6 +74,132 @@ $fontSizes: (
   xxlarge:  2rem,      // 32px
 ) !default;
 
+$formConfig: (
+  borderRadius: 5px,
+  borderWidth: 1px,
+  buttonDisabledOpacity: 0.3,
+  buttonHeight: 40px,
+  buttonHeightSmall: 35px,
+  buttonIconOffset: 25%,
+  buttonIconAfterOffset: 5px,
+  buttonMinWidth: 160px,
+  buttonMinWidthSmall: 120px,
+  circleDimension: 40px,
+  circleExternalDimension: 30px,
+  circleRadius: 20px,
+  datepickerCellSize: 38px,
+  datepickerPadding: 20px,
+  datepickerShadow: 0px 5px 10px 0px rgba(0, 0, 0, 0.10),
+  errorMargin: 5px,
+  errorPadding: 7.5px,
+  fieldDisabledOpacity: 0.65,
+  fieldHeight: 50px,
+  fieldMaxWidth: 250px,
+  fieldMaxWidthLarge: 320px,
+  fieldMaxWidthSmall: 120px,
+  fieldPadding: 0 15px,
+  fieldShadow: 0px 5px 10px 0px rgba(0, 0, 0, 0.10),
+  iconSize: 25px,
+  labelWidth: 280px,
+) !default;
+
+$tooltipConfig: (
+  arrowSize: 8px,
+  borderRadius: 5px,
+  margin: 15px,
+  padding: 10px 15px,
+  width: 320px,
+) !default;
+
+$loginConfig: (
+  borderRadius: 5px,
+  borderWidth: 1px,
+  loginShadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.10),
+  minHeight: 400px,
+) !default;
+
+$loaderConfig: (
+  borderRadius: 6px,
+  borderRadiusSmall: 4px,
+  height: 60px,
+  with: 60px,
+  heightSmall: 20px,
+  withSmall: 20px,
+) !default;
+
+$containerConfig: (
+  xsmall:  540px,
+  small:   700px,
+  medium:  960px,
+  large:   1140px,
+  xlarge:  1370px,
+) !default;
+
```

Comment:
qua dovremmo fare un controllo automatico che le chiavi di container config siano le stesse di breakpoints in modo da lanciare un errore in caso di override sbagliato